src/components/Callback.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect } from "react";
```

Comment:
When user logs in to Spotify, the API redirects the user to the `/callback` url. Here we will now have an authorization code which we then use to send to the back end.